pkg/e2e/infra/infra.go

```diff
@@ -6,9 +6,10 @@ import (
 	"strings"
 	"time"
 
+	. "github.com/onsi/ginkgo"
```

Comment:
why is this change?
If it's needed this should be in a separate commit, otherwise this introduce a lot of noise in this PR due to all the references to ginkgo/gomega making more difficult to review the real logic

Reply:
this is marked as resolved but the comment still applies